### ISSUE TITLE
Question About Waymo Dataset Download

### ISSUE THREAD

**Issue by ujaejoon** [slothfulxtx/MBPTrack3D#15]
I visited the [Waymo Open Dataset download page](https://waymo.com/open/download/) to download the dataset, but there are too many versions, and I'm not sure which one to download. From what I can tell, I should download the perception dataset rather than the motion dataset, but I'm unsure which version to choose. Could you provide guidance on this?

**Comment by Wanggggggggggggggg** [comment]
> I visited the [Waymo Open Dataset download page](https://waymo.com/open/download/) to download the dataset, but there are too many versions, and I'm not sure which one to download. From what I can tell, I should download the perception dataset rather than the motion dataset, but I'm unsure which version to choose. Could you provide guidance on this?
I am facing the same issue. May I ask if you have already resolved it?

**Comment by ujaejoon** [comment]
Nope...

**Comment by slothfulxtx** [comment]
Sorry for the late response. I remember I followed the instructions of Open3DSOT, which follows Center Point. You can find more details at [Open3DSOT](https://github.com/Ghostish/Open3DSOT?tab=readme-ov-file)

**Comment by Wanggggggggggggggg** [comment]
I noticed that in your README, the Waymo dataset seems to follow Lidar-SOT, but it appears to be using Center Point instead. Could you kindly clarify this for me? Thank you!

**Comment by slothfulxtx** [comment]
@Wanggggggggggggggg Oooooops, sorry for previous wrong reply. We don't train our network on Waymo, but use it to test the generalization ability of our method, which is different from the settings of Open3DSOT. We copy the evaluation results from STNet. Thus , you should follow [this instruction](https://github.com/fpthink/STNet).

**Comment by Wanggggggggggggggg** [comment]
@slothfulxtx Thank you for your response! May I ask whether the evaluation results are based on the validation set or the test set of the Waymo dataset? I would greatly appreciate your clarification.

**Comment by slothfulxtx** [comment]
![Image](https://github.com/user-attachments/assets/24275f8d-a9a4-4ac3-adde-4bc6bdfac0a9) It seems to be the validation set of Waymo

**Comment by Wanggggggggggggggg** [comment]
@slothfulxtx We mainly want to know whether the Waymo dataset uses the motion or perception data, and which version of the dataset was downloaded. If you could provide an answer, it would be greatly appreciated!
